hugo/content/building-the-images.md

```diff
@@ -1,3 +1,11 @@
+---
+title: "Building the Images"
+description: "Learn how to build the NGINX Gateway Fabric and NGINX images."
+weight: 300
+toc: true
+docs: "DOCS-000"
+---
+
 # Building the Images
```

Comment:
nit: Should these top level headings be removed?

Reply:
as mentioned we are still rewriting the existing docs. Content changes are not in scope of this PR

Reply:
(but yes, if you check the couple of docs where we have already worked on the content, the top level headings have been removed)